certified-connectors/Nodefusion Portal/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "6f2a7cda-d35a-40ac-b913-d3ec4066c42a",
+          "scopes": [
+            "https://3procloud.onmicrosoft.com/3proportal/user_impersonation"

```

Comment:
@hkusulja Are you sure this is a valid scope value?

Reply:
@sriyen-msft this is a valid scope for out b2c app - we have also tested it on a demo environment